lib/vm.py

```diff
@@ -100,6 +100,10 @@ def scp(self, src, dest, check=True, suppress_fingerprint_warnings=True, local_d
             local_dest=local_dest
         )
 
+    def sftp_put(self, src, dest, check=True, suppress_fingerprint_warnings=True):
+        cmd = "put {} {}".format(src, dest)
```

Comment:
Except where it makes sense to remain consistent (like in commands.py), we try to favour f-strings.
```suggestion
        cmd = f"put {src} {dest}"
```

Reply:
Fixed